### ISSUE TITLE
unpopular opinion

### ISSUE THREAD

**Issue by BDroidz** [nrlulz/ACF#385]
keep the karbine additions

**Comment by Ratakis** [comment]
No it’s not time for karbines acf I mean it’s pretty cool but I would wait until new armor was added because we are using 1940s armor against modern ammo

**Comment by BDroidz** [comment]
just use a 1x1x2 its not that hard

**Comment by BDroidz** [comment]
and ERA plates

**Comment by BDroidz** [comment]
i don't know what you mean by "new armour" but it's going to be 40's armor against modern projectiles for a LONG time.

**Comment by Ratakis** [comment]
Yes ap aphe he and heat are all modern ammo yes we used conical cheese in the 40s smoothbrain  make new armor types then add new ammo at least in RDCs sad excuse for acf he actually added different armor types.

**Comment by Ratakis** [comment]
This update seems like a weak attempt to ruin minmax. And it failed pretty hard.
